Webadmin/main.js

```diff
@@ -0,0 +1,108 @@
+
+        // Globally initializes an mqtt variable
+        const clientId = 'webadmin_' + Math.random().toString(16).substr(2, 8)
+currentNumber = 1
+const host = 'ws://ztl@ztl-mqtt.westeurope.azurecontainer.io:15675/ws'
+var serverUrl   = 'ws://ztl@ztl-mqtt.westeurope.azurecontainer.io:15675/ws';     /* wss://mqtt.cumulocity.com/mqtt for a secure connection */
+var device_name = "Webadmin";
+var username    = "ztl";
+var password    = "Ztl2019!";

```

Comment:
Shouldn't that be `topsecret`? ;-)

(in related news: you want to change that password)

Reply:
Yes but that's the way the quick and dirty push, it had to be done in a manner of short timing schedule.